scanner/fetcher.go

```diff
@@ -232,8 +231,7 @@ func (f *Fetcher) updateSTH(ctx context.Context) error {
 		quick := time.Now().Before(quickDeadline)
 		if sth.TreeSize <= lastSize || quick && sth.TreeSize < targetSize {
 			// Use an explicitly retriable error code.
-			// TODO(daviddrysdale): shift to backoff.RetriableError(fmt.Errorf(...)) when available.
-			return status.Errorf(codes.Unavailable, "wait for bigger STH than %d (last=%d, target=%d)", sth.TreeSize, lastSize, targetSize)
+			return backoff.Retry(fmt.Errorf("wait for bigger STH than %d (last=%d, target=%d)", sth.TreeSize, lastSize, targetSize))
```

Comment:
at first glance I read `backoff.Retry` as "Perform `fmt.Errorf`" multiple times.

Perhaps we should rename `backoff.Retry` to `backoff.Errorf` or something like that?

Reply:
I'd really like to make it `Errorf`-style with variadic parameter list and forward it to `fmt.Errorf` internally. Do we know if it's a good practice?

Reply:
Drat, looks like GitHub ate a comment I made on the other PR -- I suggested renaming `backoff.Retry` to `backoff.Retriable` to make things like this read better.

Reply:
But a variadic `backoff.RetriableErrorf("oh noes: %v", thingy)` would also read OK.

Reply:
```golang
package backoff
func RetriableErrorf(format string, a ...interface{}) error {
	return RetriableError(Sprintf(format, a...))
}
```